### DISCUSSION TITLE
Can not open the IPFS Desktop

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#2541] [Q&A]
GM 
---
i installed the IPFS Desktop but can not open it. the photos of app while trying to run it :
![image](https://github.com/user-attachments/assets/1d8a7791-0466-4384-9b18-989be281f95f)
![image](https://github.com/user-attachments/assets/7a77de0e-1a7c-45a2-9018-66c86869b368)
---
i tried to run it `run as administrator` too but not worked. is there any solution? i searched through the web but didn't find any solution

**Comment by EngrPips** [comment]
Hello @MostPow3rful, Is it that you can not run the IPFS Dekstop app that you installed?

**Reply by MostPow3rful** [reply]
i can not run the App.

**Reply by EngrPips** [reply]
This https://discord.gg/ipfs is an invite to their official Discord. Please join their discord and complain to them.

**Reply by MostPow3rful** [reply]
Thanks

**Reply by EngrPips** [reply]
You are welcome, Friend.
